components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -367,7 +367,8 @@ func certValidator(cm *corev1.ConfigMap, dataKey string, log logr.Logger) error
 	odhCertData, ok := cm.Data[dataKey]
 	if !ok || odhCertData == "" {
 		// Print a warning if odh-ca-bundle.crt data is empty
-		return fmt.Errorf("Warning: %s data is empty", dataKey)
+		log.Info(fmt.Sprintf("Warning: %s data is empty", dataKey))
```

Comment:
Not sure how much sense to log Info message with Warning string :slightly_smiling_face: